etc/launch-tae-logservice/cn1.toml

```diff
@@ -0,0 +1,40 @@
+# service node type, [DN|CN|LOG]
+service-type = "CN"
+
+[log]
+level = "debug"
+format = "json"
+max-size = 512
+
+
+[hakeeper-client]
+service-addresses = [
+    "127.0.0.1:32001",
+    "127.0.0.1:32011",
+    "127.0.0.1:32021",
+]
+
+[[fileservice]]
+name = "LOCAL"
+backend = "MEM"
+
+[[fileservice]]
+name = "S3"
+backend = "MEM"
+
+[[fileservice]]
+name = "ETL"
+backend = "DISK-ETL"
+data-dir = "store"
+
+[cn]
+uuid = "dd1dccb4-4d3c-41f8-b482-5251dc7a41bf"
+listen-address = "127.0.0.1:3307"
+role = "TP"
+
+[cn.Engine]
+type = "tae"
```

Comment:
"tae" means the moengine, this engine may not work with the memory DN.
dn1.toml is not necessary if you want to run moengine only.